src/devices/Sht3x/Register.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Sht3x
+{
+    /// <summary>
+    /// SHT3x Register
+    /// </summary>
+    internal enum Register : short
```

Comment:
go with the ushort - shifting short right for negative numbers does something else that what might be expected (in your case doesn't make much difference but someone might copy the project and do it by accident)

Reply:
👍Awesome! Actually, I didn't think about that in detail.